HW02_A_Graph-Fails.Rmd

```diff
@@ -1,87 +1,153 @@
-```
-
-### Graph Fail 5
-I got tired of the points, so I went to boxplots instead. However, I wanted the boxes to be all one color, but setting the color aesthetic just changed the outline? How can I make the box one color, not just the outline? 
-
-Also, the x-axis labels were overlaping, so I rotated them. But now they overlap the bottom of the graph. How can I fix this so axis labels aren't on the graph?
-
-
-```{r}
-ggplot(data = mpg, mapping = aes(x = manufacturer, y = cty, color = manufacturer)) + 
-  geom_boxplot() + 
-  theme(axis.text.x = element_text(angle = 45))
-```
-
-
-
+---
+title: "What went wrong?"
+author: "Ariana Tortolani"
+date due: "7/13/2020"
+output: github_document
+---
+
+```{r setup, include=TRUE}
+knitr::opts_chunk$set(echo = TRUE, error = TRUE)
+```
+
+## HW02 Part A
+
+In this document, I will add some examples of some coding mistakes, it is up to you to figure out why the graphs are messing up. 
+
+### First load packages
+
+It is always best to load the packages you need at the top of a script. It's another common coding formatting standard (like using the assignment operator instead of the equals sign). In this case, it helps people realize what they need to install for the script and gives an idea of what functions will be called. 
+
+It is also best coding practice to only call the packages you use, so if you use a package but end up tossing the code you use for it, then make sure to remove loading it in the first place. For example, I could use `library("tidyverse")` but since this script will only be using ggplot2, I only load ggplot2.
+
+```{r load libraries}
+library("ggplot2")
+library("magrittr") #so I can do some piping
+```
+
+
+### Graph Fail 1
+
+What error is being thrown? How do you correct it? (hint, the error message tells you)
+
+```{r}
+
+#ORIGINAL CODE WITH ERROR:
+data(mpg) #this is a dataset from the ggplot2 package
+
+mpg %>% 
+  ggplot(mapping = aes(x = city, y = hwy, color = "blue")) %>% 
+  geom_point()
+
+#EDITED CODE:
+data(mpg) #this is a dataset from the ggplot2 package
+
+  ggplot(mpg, aes(x = cty, y = hwy, color = "blue")) + 
+  geom_point()
+  
+```
+
+### Graph Fail 2
+Why aren't the points blue? It is making me blue that the points in the graph aren't blue :`(
+```{r}
+
+#ORIGINAL CODE WITH ERROR:
+ggplot(data = mpg) + 
+  geom_point(mapping = aes(x = displ, y = hwy, color = "blue"))
+
+#EDITED CODE:
+ggplot(data = mpg, aes(x = displ, y = hwy)) + 
+  geom_point(color = "blue")
+
+```
+
+
+### Graph Fail 3
+Two mistakes in this graph. First, I wanted to make the the points slightly bolder, but changing the alpha to 2 does nothing. What does alpha do and what does setting it to 2 do? What could be done instead if I want the points slightly bigger? 
+
+ANSWER: 
+Changing the alpha value changes the transparency of the points:
+  alpha = 1 : opaque 
+  alpha = 0 : transparent
+  
+  i am not sure what setting alpha = 2 does.  i can not see any changes when i generate the plot
```

Comment:
Based on the tutorial on DataCamp, setting the alpha = 2 should make the points appear at 20% transparency.